### DISCUSSION TITLE
Welcome to OpenScan2 Discussions!

### DISCUSSION THREAD

**Discussion by OpenScanEu** [OpenScanEu/OpenScan2#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sokol07** [comment]
Oh, nice, I didn't know github has got such a nice module!

**Reply by OpenScanEu** [reply]
Me neither, just found out today and enabled it in all repositories.
I am currently wrapping my head around restructuring and creating a good infrastructure.
So it's time for some trial and error again :)

**Comment by SmileyDragon** [comment]
hi,
I've just started looking at the openscan system and was interested in looking at the node-red ui page.
I start node-red with "node-red-pi --max-old-space-size=256" (without running this I get no connection to IP:1880)
ip:1880 gives me a blank node-red page with no flows
ip:1880/ui all I get is "Cannot GET /ui".
I've looked at the settings file and it points to the config files in ~/.node-red... but the files it references aren't in there. I found flows.json in Openscan/updates. I've tried importing but no local files are listed to import.
I'm at a bit of a loss being new to node-red.
Please can you tell me how to load the interface node-red so I can have a look?
thanks.

**Reply by SmileyDragon** [reply]
I just watched your youtube video and it seems that in that version the /ui..... gives you the web page and getting rid of the /ui.... gives you the node-red page.
In the version I have (Firmware: Jan 20 2022 13:58:22) just using the "192.168.1.25" gives me the web page which redirects to
"http://192.168.1.25/#!/3?socketid=EPV9tthQPlwch6o1AAAD"
and "192.168.1.25:1880" redirects me to 
"http://192.168.1.25:1880/#flow/76b547324c9854c4"
and "192.168.1.25:1880/ui" gives me the "Cannot GET /ui" error.

**Reply by SmileyDragon** [reply]
I just updated to the latest stable version and it is the same, but I now have lost the Pi info that was on the Changelog page. :(

**Reply by sokol07** [reply]
Have you tried following this guide (first step, the automatic installation from image):
https://github.com/OpenScanEu/OpenScan2/blob/main/firmware_setup.md
along with this one:
https://github.com/OpenScanEu/OpenScan2/blob/main/firmware_usage.md

In the provided RPi image NodeRED is starting by itself when the device is booted up, no need to start it manually.
I don't remember this well because I only briefly looked through configuration but IIRC the default node-red execution folder is ~/.node-red but all OpenScan files ale placed in the folder you mentioned. 
If you're following the manual installation from the first link and something is wrong - unfortunately I haven't tried it, I just used the system image provided… But we always can call Thomas, the author, to this discussion. ;)
Using the image provided you connect just to `http://openscan/` or `http://openscan/editor`, no need to use ip.

**Comment by SmileyDragon** [comment]
hi sokol07,
thanks for the links.
While I have been dowloading and making a few minor changes to the original images (during testing) I thought I had the latest info' / docs, but it turns out I was using links and documents for the old version which I saved ages ago. As soon as I used 'openscan/editor' as mentioned in the current version you linked to it opened up with all the nodes shown. :)
Thanks again.

**Comment by memento42429** [comment]
OpenScan mini has been behaving strangely since yesterday. Specifically, after you start shooting and take the specified number of shots, shooting will start again from the beginning. It won't stop.
I have tried all currently available versions, including Beta. I also tried a clean install, but the same behavior occurred.
Device: Raspberry Pi 3 Model B Rev 1.2
OS: Raspbian GNU/Linux 11 (bullseye) Jan 20 2022 13:58:22
Current version: 2024-09-09

What can I do to solve this problem? Please lend me your wisdom.

**Reply by OpenScanEu** [reply]
could you post the content of the log file (update&info tab) for further diagnostic?
it might be an issue with the limited RAM of the Pi 3B (500mb), but i am not sure about that at this point
